react/README.md

```diff
@@ -0,0 +1,124 @@
+[![Build Status](https://travis-ci.org/sparkdesignsystem/spark-design-system.svg?branch=staging)](https://travis-ci.org/sparkdesignsystem/spark-design-system)
+
+# Spark Design System React
+
+Welcome to the Spark Design System React, a product of [Quicken Loans](https://github.com/quickenloans).
+
+Spark Design System is where we collect our best solutions for how to design and build unified digital experiences for the Rocket brands.
+
+* [Spark](https://www.npmjs.com/package/@sparkdesignsystem/spark) contains the basic patterns and components that make up
+  the visual and interaction design of the Quicken Loans family of
+  Fintech products. If your application is expected to look and behave
+  like a Quicken Loans application, this package is the starting point.
+  When the Spark Sass and JS setup files are imported into an application you will receive code for all of Spark.
+
+## Getting Started
+
+**Spark assumes your project is using [Sass](https://sass-lang.com/) for styling. If you don't use Sass in your project you should follow the steps [below](#installing-sass).**
+
+### Spark React
+
+* Install the npm package.
+
+  `npm install --save-dev @sparkdesignsystem/spark @sparkdesignsystem/spark-react`
+
+* Import the Spark Sass file into the main Sass file (e.g., `App.scss`).
+
+
+  `@import '../node_modules/@sparkdesignsystem/spark/spark.scss';`
+
+  When you build and run your website, all the Spark CSS will now be bundled into your output. You can verify this by inspecting your site’s DOM and looking for the Spark CSS. There should be a `<style>` tag in the `<head>` of your page that includes Spark’s styles.
+  The Spark classes
+  are namespaced so they don't affect any existing CSS.
+
+* Now you have Spark react installed and you can import tree-shaked components like Spark Button:
+
+  `import { SprkButton } from '@sparkdesignsystem/spark-react';`
+
+  You can find code for all Spark components at react.sparkdesignsystem.com.
```

Comment:
Let's make this react.sparkdesignsystem.com a clickable link.

Reply:
Should be fixed now